scala/slick/security/scala-slick-sql-non-literal.yaml

```diff
@@ -0,0 +1,22 @@
+rules:
+- id: scala-slick-sql-non-literal
+  patterns:
+  - pattern: sql"..."
+  - pattern-regex: \#\$
+  - pattern-inside: |
+      import slick.$DEPS
+      ...
+  message: >-
+      Detected a formatted string in a SQL statement. This could lead to SQL
+      injection if variables in the SQL statement are not properly sanitized.
+      Avoid using `#$variable` and use `$variable` in `sql"..."` strings instead.
+  languages: [scala]
+  severity: WARNING
+  metadata:
```

Comment:
maybe add a `references` here to https://scala-slick.org/doc/3.3.3/sql.html#splicing-literal-values to explain why the `pattern-regex` is `\#\$`?  I am new to Scala and having the "why" attached to the rule would be useful to me as a new Scala dev.

Reply:
yep, cool idea, thanks :+1: